crates/autopilot/src/run_loop.rs

```diff
@@ -251,22 +251,11 @@ impl RunLoop {
             .iter()
             .map(|participant| participant.solution.solver().into())
             .collect::<HashSet<_>>();
-
-        let mut fee_policies = Vec::new();
-        for order_id in winning_solution.order_ids() {
-            match auction
-                .orders
-                .iter()
-                .find(|auction_order| &auction_order.uid == order_id)
-            {
-                Some(auction_order) => {
-                    fee_policies.push((auction_order.uid, auction_order.protocol_fees.clone()));
-                }
-                None => {
-                    tracing::debug!(?order_id, "order not found in auction");
-                }
-            }
-        }
+        let fee_policies = auction
```

Comment:
This PR will end up storing orders of magnitude more `fee_policies` in the DB on every successful auction. How would that reduce post-processing load?
It's important to not only read the diff but also think about what effects it will have on the remaining system.